src/FSharp.Analyzers.SDK/FSharp.Analyzers.SDK.fsproj

```diff
@@ -7,26 +7,19 @@
     <Authors>Krzysztof Cieslak</Authors>
     <Description>SDK for building custom analyzers for FSAC / F# editors</Description>
     <Copyright>Copyright 2019 Lambda Factory</Copyright>
-    <PackageLicenseExpression>MIT</PackageLicenseExpression>
```

Comment:
You will need to set `<IsPackable>true</IsPackable>` in order for that information to be picked up by the Directory.Props file

Reply:
Mmh, I see the same metadata with or without `IsPackable` 🤷